doc/ControlProtocol/ControlProtocol.tex

```diff
@@ -0,0 +1,886 @@
+% Document settings
+\documentclass[11pt]{article}
+\usepackage{fancyhdr}
+\usepackage[margin=1in]{geometry}
+\usepackage[pdftex]{graphicx}
+\usepackage{multirow}
+\usepackage{setspace}
+\pagestyle{plain}
+\usepackage{graphicx}
+\usepackage{booktabs}
+\usepackage{epstopdf}
+%\usepackage{MnSymbol,wasysym}
+\usepackage{amsmath}
+%\usepackage{mathtools}
+\usepackage{amssymb}
+\usepackage{lipsum}
+\usepackage{sympytex}
+\usepackage{siunitx}
+\setlength\parindent{0pt}
+\graphicspath{{images/}{drawings/}}
+\usepackage[acronym]{glossaries}
+
+\usepackage{rotating}
+\usepackage{array}
+\usepackage[table]{xcolor}
+
+% % % % % % % % % % % Header footer
+% % % % % % % % % % %EDIT THIS % % % % % % % % % % % % % % % % % % % %
+\pagestyle{fancy}
+\fancyhf{}
+\lhead{Imagine Project Control Protocol}
+\rhead{Alex Kneipp}
+\lfoot{RIT FIRST Alumni Association}
+\cfoot{YYYY-MM-DD}
```

Comment:
Is there a way to auto-fill this? We can write a script if necessary...

Reply:
Not really part of the MVP